lib/ProductOpener/APIProductServices.pm

```diff
@@ -382,4 +385,55 @@ sub product_services_api ($request_ref) {
 	return;
 }
 
+my %external_sources_cache = ();
+
+=head2 external_sources
+
+Get external sources but translated in target language
+
+=cut
+
+sub external_sources_api ($request_ref) {
```

Comment:
TODO @alexgarel : add a field to tell if it applies to current user based on scope. (scope_applies: bool)

Reply:
done